ui/pages/routes/routes.component.js

```diff
@@ -122,6 +96,66 @@ import {
   showOnboardingHeader,
 } from './utils';
 
+// Begin Lazy Routes
+const OnboardingFlow = React.lazy(() =>
```

Comment:
Given the volume of changes here and limited readability, is it worth listing these in alphabetical order?

Reply:
Hah, I guess it looked a little chaotic to you, but actually I was following an order.  The lazy route imports are in the same order as the `<Switch>` component.